src/stores/Binding/Store.ts

```diff
@@ -280,7 +281,9 @@ const getInitialState = (
     addEmptyBindings: (data, rehydrating) => {
         set(
             produce((state: BindingState) => {
-                const collections = state.getCollections();
+                const collections = getAllCollectionNames(
+                    state.resourceConfigs
+                );
```

Comment:
Just made a shared function to ensure these are fetched the same way every time.